### PR TITLE
Added TruckersFM

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**

### CODE REVIEW

File: TruckersFM/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "Ninja",
+    "id": "150987570715688960"
+  },
+  "service": "TruckersFM",
+  "description": {
+    "en": "TruckersFM is your staiton for driving."
```
Comment (Bas950) [inline TruckersFM/dist/metadata.json]: saiton - > station

File: TruckersFM/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+   },
+  "url": ["truckers.fm"],
```
Comment (Bas950) [inline TruckersFM/dist/metadata.json]: "url": "truckers.fm",

File: TruckersFM/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "version": "1.0",
+  "logo": "https://ninjalabs.co.uk/i/p0ezmnp.png",
+  "thumbnail": "https://ninjalabs.co.uk/i/g931v7g.png",
+  "color": "#fc0303",
+  "tags": ["TFM", "radio", "ETS2", "ATS", "Your Drive, Your Music"],
+  "category": "music",
+  "iframe": false
```
Comment (Bas950) [inline TruckersFM/dist/metadata.json]: Just remove iframe from the list

**Comment by Bas950** [comment]
Also need screenshots of the presence in work (see guidelines)
And maybe show what song is playing?

File: TruckersFM/dist/metadata.json (hunk 1 above)
Comment (Bas950) [reply TruckersFM/dist/metadata.json]: Maybe even their official description?
"The number one hit music station for Trucking Simulators. Radio made for driving, now playing online, in cab and on the go."

File: TruckersFM/dist/metadata.json (hunk 2 above)
Comment (frozaum) [reply TruckersFM/dist/metadata.json]: not really required as far as i know

Comment (OfficialCRUGG) [reply TruckersFM/dist/metadata.json]: Yup, not required. I used an array multiple times already.

Comment (Bas950) [reply TruckersFM/dist/metadata.json]: I just put it in because he has to change the other stuff, he might as well change this so it looks better.

**Comment by NinjaLabs-Dev** [comment]
Issues fixed and pushed to the branch

**Comment by frozaum** [comment]
still, we need screenshots of the presence working @NinjaLabs-Dev

**Comment by NinjaLabs-Dev** [comment]
@frozaum How would I do that, I've looks and can't find anything in the documentation.

**Comment by Bas950** [comment]
@NinjaLabs-Dev shift when you have the extension open and it will say load presence, then click the dist folder of the presence.

**Comment by NinjaLabs-Dev** [comment]
@Bas950 Got it thanks, just playing around with different configs right now. Is it possible to change the "Playing" to "Listening to"

**Comment by Bas950** [comment]
Is it possible to change the "Playing" to "Listening to"
No

**Comment by NinjaLabs-Dev** [comment]
DiscordCanary_lJ6rxuQ3qD
DiscordCanary_N4JkLW68hB
Here are the screenshots you requested.

**Pull request merged**